src/test/java/org/gridsuite/study/server/StudyServiceTest.java

```diff
@@ -55,14 +54,16 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 import static org.mockito.Mockito.when;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.head;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
 @RunWith(SpringRunner.class)
 @AutoConfigureMockMvc
 @SpringBootTest
 @DisableElasticsearch
 @ContextConfigurationWithTestChannel
+@DirtiesContext
```

Comment:
Because the tests modify the ApplicationContext by doing
```java
caseService.setCaseServerBaseUri(wireMockServer.baseUrl());
networkConversionService.setNetworkConversionServerBaseUri(wireMockServer.baseUrl());
userAdminService.setUserAdminServerBaseUri(wireMockServer.baseUrl());
```
so we need to say to Spring-Boot that the context has been modified.